gulpfile.babel.js

```diff
@@ -0,0 +1,105 @@
+import gulp from 'gulp';
+import util from 'gulp-util';
+import minimist from 'minimist';
+import rename from 'gulp-rename';
+import replace from 'gulp-replace';
+import path from 'path';
+import del from 'del';
+import coffee from 'gulp-coffee';
+import stream from 'vinyl-source-stream';
+import browserify from 'browserify';
+import coffeeify from 'coffeeify';
+import shim from 'browserify-shim';
+import { Server as Karma } from 'karma';
+
+import { version } from './package';
+import { standalone } from './package';
+import { filename } from './package';
```

Comment:
You could consolidate these into a single line with the following

`import {  version, standalone, filename } from './package'`